docs/first-order/test_2p_responses.ipynb

```diff
@@ -0,0 +1,1092 @@
+{
```

Comment:
<p>You should show the raw data to visualize the repeats</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>